Makefile

```diff
@@ -148,6 +148,14 @@ distclean:
 install: checkprefix nvim
 	+$(BUILD_TOOL) -C build install
 
+clang-tidy: build/.ran-cmake
+	@test $(FILE) \
+		&& clang-tidy --config-file .clang-tidy -p build $(FILE) \
+		|| $(MAKE) clang-tidy-full
+
+clang-tidy-full: build/.ran-cmake
+	+$(BUILD_TOOL) -C build clang-tidy-full
```

Comment:
You can replace this by adding `clang-tidy-full` to the list with `lintlua lintsh lintuncrustify ...`

Reply:
Actually, is a make target to run `clang-tidy-full` even needed? There's no way anyone will run this locally since it takes such a long time so I don't think there's much point.

Reply:
I just wanted to follow the steps of clint, but personally I don't think this specific target offers all that much.